website/content/zh/build/images/arrow.svg

```diff
@@ -0,0 +1,12 @@
+<!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In  -->
```

Comment:
@matthewcarleton can we move these images to a language neutral location like https://github.com/operator-framework/operator-sdk/tree/master/website/static ?

Reply:
I don't see why not. Do you want me to do it?

Reply:
Done. See comment below.